src/main/java/com/github/markusbernhardt/seleniumlibrary/CustomRobotDriverElement.java

```diff
@@ -0,0 +1,48 @@
+package com.github.markusbernhardt.seleniumlibrary;
+
+import com.github.markusbernhardt.seleniumlibrary.keywords.BrowserManagement;
+import com.github.markusbernhardt.seleniumlibrary.keywords.Robot;
+import org.openqa.selenium.WebDriver;
+
+import javax.script.ScriptEngine;
+import javax.script.ScriptEngineManager;
+import javax.script.ScriptException;
+import java.lang.reflect.Field;
+
+/**
+ * For extends of custom keywords of WebDriver actions
+ */
+public class CustomRobotDriverElement {
+
+    private static SeleniumLibrary s;
+    private static BrowserManagement b;
+    private Robot robot = new Robot();
+
+    public CustomRobotDriverElement() throws NoSuchFieldException, IllegalAccessException {
+        try {
+            CustomRobotDriverElement.s = getLibraryInstance();
+        } catch (javax.script.ScriptException e) {
+            e.printStackTrace();
```

Comment:
It seems like we need to do return here at least.

Also, could we log the exception with some logger? Because in the case with stdout we won't be able to turn the logging off.

What do you think?

Reply:
You are talking about some logging for the exception?

Reply:
Yes, it's related to this particular line.

Reply:
Added SeleniumLibraryNonFatalException to catch block in the CustomRobotDriverElement constr.
What do you think about it? I cannot use the default logging (@Autowired protected Logging logging;) of the framework in the class.

Reply:
> What do you think about it?

It's much better to me. Thank you!

> I cannot use the default logging (@Autowired protected Logging logging;) of the framework in the class.

Yes, I understand it now. Thank you!